src/api/app/models/bs_request_action_submit.rb

```diff
@@ -170,6 +170,16 @@ def forward
     forward_object
   end
 
+  def source_srcmd5
+    source_package = Package.find_by_project_and_name(source_project, source_package)
+    source_package.dir_hash({ rev: source_rev }.compact)['srcmd5']
+  end
+
+  def target_srcmd5
+    target_package = Package.find_by_project_and_name(target_project, target_package)
+    target_package.dir_hash['srcmd5']
+  end
```

Comment:
```suggestion
  def source_srcmd5
    Package.find_by_project_and_name(source_project, source_package).dir_hash({ rev: source_rev }.compact)['srcmd5']
  end

  def target_srcmd5
    Package.find_by_project_and_name(target_project, target_package).dir_hash['srcmd5']
  end
```

I would prefer not using local variables (`source_package` and `target_package`) with the same name as fields of the Active Record model.